Dockerfile

```diff
@@ -0,0 +1,23 @@
+﻿FROM mcr.microsoft.com/dotnet/sdk:5.0-alpine
+
+EXPOSE 5000
+EXPOSE 5001
+
+ARG USER=dev
+ARG DUID=1000
+ARG DGID=1000
+
+VOLUME [ "/src" ]
+WORKDIR /src
+
+RUN addgroup -g ${DGID} -S ${USER}
+RUN adduser -S -G ${USER} -u ${DUID} -s /bin/bash ${USER}
+USER ${USER}
```

Comment:
I had to remove the group create operation in order to make things work.
It might be related to macOS differences.

```dockerfile
# RUN addgroup -g ${DGID} -S ${USER}
RUN adduser -S -u ${DUID} -s /bin/bash ${USER}
```

Reply:
Updated the makefile, group creation will be skipped on mac, also the platform now fixed for development, while the publish image is multiarch